scriptworker/version.py

```diff
@@ -50,7 +50,7 @@ def get_version_string(version):
 
 # 1}}}
 # Semantic versioning 2.0.0  http://semver.org/
-__version__ = (3, 0, 0)
+__version__ = (3, 1, 0)
```

Comment:
based on list of changes and chosen versioning semantics, I suppose to me the change could be either a second or third digit bump: e.g. 3.0.1 potentially. I guess it depends on how much of a change something like "allowlists updates" are